lib/smart_answer_flows/register-a-birth/embassy_result.govspeak.erb

```diff
@@ -0,0 +1,265 @@
+<% content_for :body do %>
+$!You can apply to register the birth with the <%= embassy_high_commission_or_consulate %> in <%= registration_country_name_lowercase_prefix %>.$!
+
+<% if %w(usa).include?(country_of_birth) %>
```

Comment:
Yes. I've seen this pattern in a lot of places. I think I'd prefer to keep it like this in this PR to make it more obvious where the condition code as come from.